.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Is this because of mixing Husky versions? There should just be .husky/pre-commit so I'm wondering how this directory got created for you?